data/core/macros/abilities.cfg

```diff
@@ -481,6 +481,21 @@ Enemy units cannot see this unit while it is in deep water, except if they have
     [/chance_to_hit]
 #enddef
 
+#define WEAPON_SPECIAL_DEFLECT
+    # Canned definition of the Deflect ability to be included in a
+    # [specials] clause.
+    [chance_to_hit]
+        id=deflect
+        name= _ "deflect"
+        description= _ "When used defensively, this attack reduces the opponent's chance to hit by 10%."
+        special_note={INTERNAL:SPECIAL_NOTES_DEFLECT}
```

Comment:
The `INTERNAL` macros are just a deprecation pathway to avoid writing the strings out twice. For new abilities, you should put the string here where it's used.

Reply:
OK, I was just following all the other notes, but I think I understand.